WalletWasabi.Fluent/Views/Shell/Shell.axaml

```diff
@@ -15,7 +15,53 @@
   </i:Interaction.Behaviors>
   <Panel>
     <shell:Background />
-    <shell:MainScreen x:CompileBindings="False" IsHitTestVisible="{Binding $parent.DataContext.IsMainContentEnabled^, FallbackValue=True}" />
+    <shell:MainScreen x:CompileBindings="False"
+                      IsHitTestVisible="{Binding $parent.DataContext.IsMainContentEnabled^, FallbackValue=True}">
+      <shell:MainScreen.Styles>
+        <Style Selector="shell|MainScreen[IsHitTestVisible=True]">
```

Comment:
This gets executed when the window appears on opening the application. Can it be somehow skipped?